west.yml

```diff
@@ -53,7 +53,7 @@ manifest:
       revision: a12d92816a53a521d79cefcf5c38b9dc8a4fed6e
       path: modules/hal/cypress
     - name: hal_nordic
-      revision: 1ebfdf148b2ca1d9ab9267ab2cd64cc00f32d505
+      revision: pull/37/head
```

Comment:
Should this be changed in the SoC commit instead?

Reply:
Yes, I will change it to proper hash when https://github.com/zephyrproject-rtos/hal_nordic/pull/37 is merged

Reply:
I guess the comment here is to do the change in the commit that contributes the SoC definition :)